src/Discovery/Discovery.css

```diff
@@ -518,3 +518,28 @@
   color: #313839;
   font-weight: normal;
 }
+
+.discovery-modal__field {
+  display: flex;
+  justify-content: space-between;
+  padding: 2px 0px;
+}
+
+.discovery-modal__fieldlabel {
+  padding-right: 10px;
+}
+
+.discovery-modal__subheading {
+  background-color: #dfe3e2;
+  border-radius: 4px;
+  padding: 0px 10px;
+}
+
+.discovery-modal__fieldgroup {
+  padding: 4px;
+}
+
+.discovery-modal__tagsfield {
+  display: flex;
+  justify-content: end;
+}
```

Comment:
I think we should update the CSS to let the activated Tab title font color honors `--g3-color__base-blue` as baseline color, and we can override that in gitops.css if needed